ios/Classes/JitsiViewController.swift

```diff
@@ -1,7 +1,15 @@
 import UIKit
 import JitsiMeet
 
-class JitsiViewController: UIViewController {
+class JitsiViewController: UIViewController,PiPViewCoordinatorDelegate {
+    
+    func exitPictureInPicture() {
```

Comment:
Could  this function be moved to the same extension class as the enterPictureInPicture function?